src/Scan.js

```diff
@@ -134,6 +96,57 @@ class Scan extends React.Component {
     }
   };
 
+  static manifest = Object.freeze({
```

Comment:
Usually, static props are grouped together. So I think it would be better to keep the previous way when manifest comes right at the top before the `propTypes` or vice versa when `manifest` comes after `propTypes`. What do you think?

Reply:
I moved it because of eslint warning, so I don't know if your solution will work in this case.

Reply:
Got it, let me try it locally, I will circle back shortly.

Reply:
@zburke, @ryandberger, I see that this rule is because of react/sort-comp prop which is appeared after the update. I think that this is ridiculous. As result I see the three way to solve it: 
1. either disable it locally;
2. disable it globally;
3. move static props out of the component.

Thoughts?

Reply:
I have seen this crop up in other repos since the upgrade so I think a global solution is appropriate. There is already a definition here: https://github.com/folio-org/eslint-config-stripes/blob/master/index.js#L92 so I will need to see how it should be adjusted to allow our established pattern to persist.

Reply:
Okay, I see from https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-comp.md that `static-variables` was a recent addition in considering placement. I can update the global config to accommodate our patterns.

Reply:
Please go ahead, it would improve stuff.

Reply:
PR here: https://github.com/folio-org/eslint-config-stripes/pull/56